include/openmc/source.h

```diff
@@ -73,27 +101,25 @@ class IndependentSource : public Source {
 
   // Properties
   ParticleType particle_type() const { return particle_; }
-  double strength() const override { return strength_; }
 
   // Make observing pointers available
   SpatialDistribution* space() const { return space_.get(); }
   UnitSphereDistribution* angle() const { return angle_.get(); }
   Distribution* energy() const { return energy_.get(); }
   Distribution* time() const { return time_.get(); }
 
-private:
-  // Domain types
-  enum class DomainType { UNIVERSE, MATERIAL, CELL };
+protected:
+  // Note that this method is not used since IndependentSource directly
+  // overrides the sample method
+  SourceSite sample_no_rejection(uint64_t* seed) const override { return {}; }
```

Comment:
Oh OK! I see what sample no rejection does. This is the proposal distribution. The first thing that came to mind for me reading "no_rejection" is that this implements some other sampling method that avoids using rejection. Instead it's the proposal distribution inside a rejection loop. I'd suggest renaming to "sample_proposal" or similar.

This should definitely be documented in the base class with a brief explainer on how it interacts with child classes.